core-scroll-threshold.html

```diff
@@ -125,14 +136,22 @@
       this._boundScrollHandler = this.checkThreshold.bind(this);
     },
 
+    detached: function() {
+      // Remove listener for any previous scroll target
+      if (this._scrollTarget) {
+        this._scrollTarget.removeEventListener(this._boundScrollHandler);
```

Comment:
`this._scrollTarget.removeEventListener('scroll', this._boundScrollHandler);`
?